cypress/e2e/po/pages/about.po.ts

```diff
@@ -0,0 +1,52 @@
+import PagePo from '@/cypress/e2e/po/pages/page.po';
+
+/**
+ * About page
+ */
+export default class AboutPagePo extends PagePo {
+  static url = '/about'
+
+  constructor(pageUrl = AboutPagePo.url) {
+    AboutPagePo.url = pageUrl;
+    super(AboutPagePo.url);
+  }
+
+  diagnosticsBtn() {
+    return cy.getId('about__diagnostics_button');
+  }
+
+  links(value: string): Cypress.Chainable {
+    return this.self().get('a').contains(value);
+  }
+
+  getLinkDestination(value: string): Cypress.Chainable {
+    return this.links(value).invoke('prop', 'href');
+  }
+
+  clickVersionLink(value: string) {
+    this.links(value).then((el) => {
```

Comment:
```
    this.links(value)
        .then((el) => { expect(el).to.have.attr('target');  })
        .invoke('removeAttr', 'target')
        .click();
```

Reply:
Linter is throwing error when I use the above format https://github.com/rancher/dashboard/actions/runs/6189189193/job/16802779417?pr=9667